claripy/backends/backend_concrete.py

```diff
@@ -33,6 +35,12 @@ def BVV(value, size):
             raise BackendError("can't handle empty BVVs")
         return bv.BVV(value, size)
 
+    @staticmethod
+    def StringV(value, size):
+        if value is None:
```

Comment:
Does it handle the case where `value == ""`? If not, you might want to do `if not value:` instead.

Reply:
True! Fixed